tests/commons/j2/test_j2_functions.py

```
@@ -30,10 +30,14 @@ def assertFormat(
     )
 
 
-def assertCase(short_name: str, case: tuple) -> None:
+def assertCase(
+    short_name: str, case: tuple, pfx_variants: list[str] | None = None
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)